docs/spec/v1/kustomizations.md

```diff
@@ -169,6 +169,39 @@ kustomize.toolkit.fluxcd.io/prune: disabled
 For details on how the controller tracks Kubernetes objects and determines what
 to garbage collect, see [`.status.inventory`](#inventory).
 
+### Deletion policy
+
+`.spec.deletionPolicy` is an optional field that is allowing control over
+garbage collection when a Kustomization object is deleted. The default behavior
+is to mirror the value of [`.spec.prune`](#prune).
```

Comment:
Not totally sure about this one, but it's not the concrete true/false value that is mirrored. Should that be addressed here?

```suggestion
is to mirror the configuration of [`.spec.prune`](#prune).
```